axonivy-engine/11.4/Dockerfile

```diff
@@ -10,7 +10,6 @@ RUN apt-get update && \
     mkdir /tmp/ivy/data && \
     mkdir /tmp/ivy/logs && \
     mkdir /tmp/ivy/configuration/applications && \
-    chown -R 1000:0 /tmp/ivy && \
     chmod -R g=u /tmp/ivy
```

Comment:
This is needed?

Reply:
Later there is an ADD where --chown is correctly done.